apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -375,6 +375,29 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
             if (unitType) return unitType
         }).filter(Boolean)
     }, [initialHasAllProperties, initialNewsItemScopes, initialProperties.length, initialSectionKeys])
+    const commonTemplates = useMemo(() => {
+        const commonTemplates = {}
+
+        for (const id in templates) {
+            if (templates[id].type === NEWS_TYPE_COMMON || isNull(templates[id].type)) {
+                commonTemplates[id] = templates[id]
+            }
+        }
+
+        return commonTemplates
```

Comment:
Why not just filter the `templates` array?
```
const commonTemplates = useMemo(() => templates.filter(({type})=>type === NEWS_TYPE_COMMON), [...])
```

Why not fill both arrays in the same loop?

Or you may filter the templates list within the news type change handler

Reply:
because its not array but an object of objects, I can change it to array but dont know why that would be better

its indeed possible to filter everything in one loop but I suppose that two loops for each type would be more readable

I also thought it wouldnt make sense to filter every time you change type since its potentially extra operation